docs/flow/modin/backends/pandas/parsers.rst

```diff
@@ -0,0 +1,21 @@
+:orphan:
+
+Pandas Parsers Module Description
+"""""""""""""""""""""""""""""""""
+High-Level Module Overview
+''''''''''''''''''''''''''
+
+This module houses parser classes (classes that are used for data parsing on the workers) and util functions for handling parsing results. `PandasParser` is base class for parser classes with Pandas backend, that contains methods common for all child classes. Other module classes implement `parse` function that performs parsing of specific format data basing on the chunk information computed in the `modin.engines.base.io` module. After chunk data parsing is completed, resulting `DataFrame`-s will be splitted into smaller `DataFrames`-s according to `num_splits` parameter, data type and number or rows/columns in the parsed chunk, and then these frames and some additional metadata will be returned.
```

Comment:
```suggestion
This module houses parser classes (classes that are used for data parsing on the workers) and util functions for handling parsing results. `PandasParser` is base class for parser classes with pandas backend, that contains methods common for all child classes. Other module classes implement `parse` function that performs parsing of specific format data basing on the chunk information computed in the `modin.engines.base.io` module. After chunk data parsing is completed, resulting `DataFrame`-s will be splitted into smaller `DataFrame(s)` according to `num_splits` parameter, data type and number or rows/columns in the parsed chunk, and then these frames and some additional metadata will be returned.
```

Reply:
This suggestion conflicts with @vnlitvinov [suggestion](https://github.com/modin-project/modin/pull/2952#discussion_r609358647) regarding `DataFrame(s)` change. Which one we would prefer?

Reply:
There was double "s" in the word. As for using `DataFrame(s)` or `DataFrame`-s, I don't have specific preferences on it. Just let's keep it uniform everywhere :)

Reply:
Right, double `s` should be removed, so let's make `DataFrame`-s )

Reply:
Okay :)